src/main/resources/assets/minecolonies/lang/en_us.json

```diff
@@ -638,102 +635,102 @@
+  "com.minecolonies.coremod.job.guard.toolclickguardtoofar": "Too far away from Guard Tower, please upgrade your Guard Tower to set this spot!",
+  "com.minecolonies.coremod.saturation.0": "I'm starving, I can't work anymore!",
+  "com.minecolonies.coremod.saturation.3": "I'm quite hungry, I feel very unproductive!",
+  "com.minecolonies.coremod.saturation.5": "I'm hungry, I'm not that productive anymore.",
   "com.minecolonies.coremod.saturation.7": "I'm quite saturated, but I'll have to eat again soon",
 
   "item.minecolonies.scepterbeekeeper.removehive": "Hive removed successfully",
   "item.minecolonies.scepterbeekeeper.addhive": "Hive added successfully",
-  "item.minecolonies.scepterbeekeeper.maxhives": "You have reached the maximum number of hives. Upgrade the beekeeper to increase the number they can work",
+  "item.minecolonies.scepterbeekeeper.maxhives": "You have reached the maximum number of hives. Upgrade the Apiary to increase the max number.",
 
-  "com.minecolonies.coremod.item.permissionscepter.setmode": "Set to mode '%s'",
-  "com.minecolonies.coremod.item.permissionscepter.permission.deny": "You do not have permission to edit blocks with this tool.",
-  "com.minecolonies.coremod.item.permissionscepter.addblock.success": "Added block type '%s'",
-  "com.minecolonies.coremod.item.permissionscepter.addposition.success": "Added location '%d %d %d'",
-  "com.minecolonies.coremod.item.permissionscepter.removeblock.success": "Removed block '%s'",
-  "com.minecolonies.coremod.item.permissionscepter.removelocation.success": "Removed location '%d %d %d'",
-  "tile.blockhut.warehouse.limit": "The amount of warehouses is limited to 1 per colony.",
-  "tile.blockhut.tavern.limit": "The amount of Taverns is limited to 1 per colony.",
+  "com.minecolonies.coremod.item.permissionscepter.setmode": "Set to mode %s.",
+  "com.minecolonies.coremod.item.permissionscepter.permission.deny": "You do not have permission to edit blocks with this tool!",
+  "com.minecolonies.coremod.item.permissionscepter.addblock.success": "Added block type %s.",
+  "com.minecolonies.coremod.item.permissionscepter.addposition.success": "Added location %d %d %d.",
+  "com.minecolonies.coremod.item.permissionscepter.removeblock.success": "Removed block %s.",
+  "com.minecolonies.coremod.item.permissionscepter.removelocation.success": "Removed location %d %d %d.",
+  "tile.blockhut.warehouse.limit": "The amount of Warehouses is limited to 1 per colony. You cannot place any more in this colony.",
+  "tile.blockhut.tavern.limit": "The amount of Taverns is limited to 1 per colony. You cannot place any more in this colony.",
 
-  "com.minecolonies.coremod.gui.workerhuts.baker": "Baker",
-  "com.minecolonies.coremod.bakery.nofurnace": "My building doesn't have any furnaces, please repair it.",
+  "com.minecolonies.coremod.gui.workerhuts.baker": "Bakery",
+  "com.minecolonies.coremod.bakery.nofurnace": "My Bakery doesn't have any furnaces, please repair it!",
```

Comment:
We can't make this change. This message is used by almost all the smeltercrafters, so you'll get it from stonesmelter, glassblower, dyer and soon the cook's assistant.

Reply:
Ah, I thought it was just the Bakery because the thing before it (not sure what that's called) said Bakery. Fixing now!